aws/s3.tf

```diff
@@ -123,3 +123,94 @@ module "s3_bucket" {
 
   tags = local.tags
 }
+# aws S3 backup //////////////////////
+resource "aws_backup_vault" "datagrok_public_vault" {
+  name = var.s3_backup_vault_name
+}
+
+resource "aws_iam_role" "backup_role" {
+  name                = "s3-backup-role"
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect = "Allow"
+        Principal = {
+          Service = "backup.amazonaws.com"
+        }
+        Action = "sts:AssumeRole"
+      }
+    ]
+  })
+}
+
+resource "aws_iam_policy" "s3_backup" {
+  name        = "backup-s3"
+  description = "policy for backup s3 datagrok-public bucket"
```

Comment:
datagrok-public is a hardcoded name. It is a name for our bucket and won't be the same for clients.
You should use `s3_name` here instead